daemon/src/monitor.rs

```diff
@@ -110,7 +111,7 @@ where
                             actor.monitor_refund_finality(&params,cfd.order.id);
                         }
                         CetStatus::OracleSigned(attestation) => {
-                            actor.monitor_cet_finality(map_cets(dlc.cets, dlc.address.script_pubkey()), attestation, cfd.order.id)?;
+                            actor.monitor_cet_finality(map_cets(dlc.cets, dlc.maker_address.script_pubkey()), attestation, cfd.order.id)?;
```

Comment:
Is this line what https://github.com/comit-network/hermes/issues/238 is referring to? Can we just fix this by monitoring for both here?

Reply:
Yes it is. I'd think this should work.